tslint.json

```diff
@@ -1,5 +1,5 @@
 {
-  // TODO: no "../" in imports
   // TODO: no unused imports
   // TODO: multiple imports { foo, bar } on seperate lines
```

Comment:
Still, do not like this enforcement :P

Reply:
I definitely agree with that, will remove.